webportal/webapp/src/environments/environment.dev-private.ts

```diff
@@ -0,0 +1,13 @@
+export const environment = {
+  production: false,
+  STREAMING_URL: 'https://dev-stream.securedatacommons.com/guacamole/?authToken=',
+  REGION: 'us-east-1',
+  USER_POOL_ID: 'us-east-1_Y5JI7ysvY',
+  CLIENT_ID: '22gr1lfintehcaac2cdmmsur52',
+  IDENTITY_PROVIDER: 'dev-dot-ad',
+  APP_DOMAIN: 'dev-sdc-dot-webportal',
+  REDIRECT_URL: 'https://dev-proxy-portal.securedatacommons.com/index.html', //dev environment
+  API_ENDPOINT: 'https://dev-proxy-portal.securedatacommons.com/webportal-api/',
+  LOGIN_GOV_COGNITO_APP_CLIENT_ID: '3ikuiqen3fdfl5brs5uk49cvsu',
+  LOGIN_GOV_ACCOUNT_LINK_URL: 'https://dev-proxy-portal.securedatacommons.com',
```

Comment:
I found this name to be pretty confusing, because it isn't using `login-gov-proxy` but rather `account-link`. The prefix made me think it was the other API, and account-link doesn't actually have the `login-gov` prefix in its API.

Reply:
I'd be down with changing it. It is confusing because it deals with Cognito linking, not Login.gov specifically.